dashboards-observability/common/constants/shared.ts

```diff
@@ -26,6 +26,7 @@ export const observabilityTitle = 'Observability';
 export const observabilityPluginOrder = 6000;
 
 // Shared Constants
+export const SQL_DOCUMENTATION_URL ='https://opensearch.org/docs/latest/observability-plugins/ppl/commands/'
```

Comment:
I think these links are out of date, when I visit that URL it redirects me to the OpenSearch documentation home page

Reply:
Thanks for the catch will update this!

Reply:
I think the observability link on website is changed from `observability-plugin to observbility ` 🙅🏽

Reply:
this is the PPL link, SQL should be https://opensearch.org/docs/latest/search-plugins/sql/index/

Reply:
I have pinged Ketih for the website link change. If the name change was intentional I'll change all the links in observability.

Reply:
updated all links & fixed sql link